docs/guides/cluseter-admin.md

```diff
@@ -0,0 +1,31 @@
+---
```

Comment:
@nirarg can you review this to determine if this guide satisfies https://issues.redhat.com/browse/OSDOCS-7474

I'll add a guide in the ROSA docs after this and the landing page merge.